__test__/GeneExpressionTSnePlot.test.js

```diff
@@ -154,4 +155,14 @@ describe(`GeneExpressionTSnePlot`, () => {
     expect(tree).toMatchSnapshot()
   })
 
+  test(`contains no atlas autocomplete control when flag is false`, () => {
+    const randomSeries = randomHighchartsSeriesWithSeed()
+    const onSelectGeneId = () => {}
+
+    const wrapper = mount(<GeneExpressionTSnePlot height={600} expressionGradientColours={gradientColourRanges()} atlasUrl={``} suggesterEndpoint={``} onSelectGeneId={onSelectGeneId} loading={true} plotData={plotData(randomSeries)} highlightClusters={[]} speciesName={``} showControls={false}/>)
+
+
+    expect(wrapper.find(AtlasAutocomplete).length).toBe(0)
```

Comment:
Please add a test for the existence of `AtlasAutocomplete` with default `showControls` props. Remove the extra spaces.

Reply:
Good point. Added new test case for this